lib/cldr/ldml/attributes.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Cldr
+  module Ldml
+    COMMENT_RE = /^\s*<!--@(.*)-->\s*$/.freeze
```

Comment:
It wasn't intentional. Thanks for pointing it out.

Actually, `COMMENT_RE` doesn't need `freeze`, since [regex literals are always frozen](https://bugs.ruby-lang.org/issues/16377).

I've removed `freeze` from `COMMENT_RE`.